ddtrace/opentracer/span_context.py

```diff
@@ -1,15 +1,39 @@
 from opentracing import SpanContext as OpenTracingSpanContext
 
 
+from ddtrace.context import Context
+
+
 class SpanContext(OpenTracingSpanContext):
-    """"""
+    """Implementation of the OpenTracing span context."""
+
+    def __init__(self, trace_id=None, span_id=None, sampled=True,
+                 sampling_priority=None, baggage=None, context=None):
+
+        baggage = baggage or OpenTracingSpanContext.EMPTY_BAGGAGE
```

Comment:
`OpenTracingSpanContext.EMPTY_BAGGAGE ` is mutable (https://github.com/opentracing/opentracing-python/blob/master/opentracing/span.py#L30) so I don't think we're solving the problem. I would say to simply initialize a new dict (`{}`). Unless there is a change in OT, we can't use that data structure because it will be modified globally.

Reply:
Ugh! My bad. I’ll correct this and add a test to ensure that the problem doesn’t prop up in the future (or the present).